boltdesignsystem/bolt
# Remove 'brand-' prefix from icon in _icon-banner.twig

## Jira

http://vjira2:8080/browse/BDS-2052

## Summary

Removes 'brand-' prefix from icon in `_icon-banner.twig`.

## Details

So that `_icon-banner.twig` can accept icons that don't begin with 'brand-', remove the hardcoded prefix.

## How to test

- Review code changes
- Verify icon banners are still appearing in cards on feature branch.
  - See cards on Dashboard, Search results, and look for the icons that look like this:
![image](https://user-images.githubusercontent.com/36336/75788984-ffd52f80-5d36-11ea-92b8-94226d29ea51.png)
